### ISSUE TITLE
problem in getting the output of an intermediate layer in COnvolutional neural network

### ISSUE THREAD

**Issue by chanchalIITP** [chanchalIITP/Convolutional-support-vector-machine#1]
My code is like 
model = Sequential()
model.add(ZeroPadding2D((1, 1), input_shape=(1, 10, 4), dim_ordering='tf'))
model.add(Convolution2D(4, 1, 1, activation='relu'))
model.add(ZeroPadding2D((1, 1), dim_ordering='tf'))
model.add(MaxPooling2D((2, 2), strides=(2, 2)))
model.add(Flatten())
model.add(Dense(2, activation='softmax'))
sgd = SGD(lr=1e-2, decay=1e-6, momentum=0.9, nesterov=False)
model.compile(optimizer='adam',loss='binary_crossentropy', metrics=['accuracy'])

I want to get the output of max pooling layer . What should i write for it. Please anyone help me .

**Comment by chanchalIITP** [comment]
#2495

**Comment by chanchalIITP** [comment]
@stevenhanjun ... I think you may help me
